client/packages/common/src/ui/components/modals/InputModal/InputModal.tsx

```diff
@@ -21,7 +21,11 @@ export const InputModal = ({
   onClose,
   title,
 }: InputModalProps) => {
-  const { Modal } = useDialog({ isOpen, onClose });
+  const { Modal } = useDialog({
+    isOpen,
+    onClose,
+    disableMobileFullScreen: true,
+  });
```

Comment:
Input modal is small (used for like tax input) so don't need the full screen capability here